omod/src/main/webapp/resources/htmlforms/HMIS-005-IntegratedAntenatalRegister.xml

```diff
@@ -1,500 +1,735 @@
+                                            </div>
+                                            <div class="col-md-3">
+                                                <obs conceptId="165383" labelText="Sickle Cell" />
+                                            </div>
+                                        </div>
+                                        </div>
+                                        </div>
 
+                                        <div class="card">
+                                    <div class="card-header">
+                                        Urine
+                                    </div>
+                                    <div class="card-body">
+                                    <div class="row">
+                                            
+                                            
+                                            <div class="col-md-6">
+                                                <obs conceptId="717" labelText="Protein" />
+                                            </div>
+                                            <div class="col-md-6">
+                                                <obs conceptId="887" labelText="Glucose" />
+                                            </div>
+                                        </div>
+                                    </div>
+                                    </div>
+
+                                    <div class="card">
+                                        <div class="card-header">
+                                            Women Syphilis and Hep B Test Results
+                                        </div>
+                                        <div class="card-body">
+                                            <div class="row">  
+                                            <div class="col-md-6">
+                                                    <obs conceptId="299" labelText="Syphillis" id="syphillis-woman" />
                                                 </div>
+                                                <div class="col-md-6">
+                                                    <obs conceptId="1321" labelText="Hepatitis B" id="hepatitis-b-woman" />
+                                                </div>
+                                            </div>
+                                        </div>
+                                    </div>
 
-                                                <div class="col-md-3">
-                                                    <obs conceptId="99767" labelText="Referral Number" />
+                                    <div class="card">
+                                        <div class="card-header">
+                                            Partner Syphilis and Hep B Test Results
+                                        </div>
+                             
```

Comment:
@semujju please move this field to the investigations tab.

Reply:
Done